homeassistant/components/weatherflow/sensor.py

```diff
@@ -371,14 +372,15 @@ def last_reset(self) -> datetime | None:
             return self.device.last_report
         return None
 
-    @property
-    def native_value(self) -> datetime | StateType:
-        """Return the state of the sensor."""
-        return self.entity_description.get_native_value(self.device)
+    def _update_state(self) -> None:
```

Comment:
```suggestion
    def _async_update_state(self) -> None:
```

Since we have both sync and async code in this code base we usually add `async` in the function name to indicate its safe to run in the event loop and will not do blocking I/O